modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/problem/ObjectValidationRuleEngineProblemMapper.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.rest.internal.vulcan.problem;
+
+import com.liferay.object.exception.ObjectValidationRuleEngineException;
+import com.liferay.object.validation.rule.ObjectValidationRuleResult;
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONFactory;
+import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.util.ListUtil;
+import com.liferay.portal.vulcan.problem.Problem;
+import com.liferay.portal.vulcan.problem.ProblemMapper;
+
+import java.util.List;
+import java.util.Locale;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Luis Miguel Barcos
+ */
+@Component(service = ProblemMapper.class)
+public class ObjectValidationRuleEngineProblemMapper
```

Comment:
I think this should be renamed from `ObjectValidationRuleEngineProblemMapper` to `ObjectValidationRuleEngineExceptionProblemMapper`

Reply:
Done.